index.Rmd

```diff
@@ -261,7 +261,7 @@ income_bracket[2,1] = "Between 20K and 40k"
 income_bracket[3,1] = "Between 40k and 60k"
 income_bracket[4,1] = "More than 60k"
 
-income_dists <- data.frame(matrix(ncol = 1, nrow = 4))
+income_dists <- data.frame(matrix(ncol = 1, nrow = 4)) 
```

Comment:
In R, we can merge two data frames by using rbind()/merge(). Then we don't have to input each element into matrix by specifying its location. Therefore, I think you might want to use data frame instead of matrix here. 
Also, setting 1609.344 as a variable might be a good idea as well. (Just in case there is any typo)